db/db_mysql/migrations/20160118194630_init.sql

```diff
@@ -2,6 +2,7 @@
 -- +goose Up
 -- SQL in section 'Up' is executed when this migration is applied
 CREATE TABLE IF NOT EXISTS users (id integer primary key auto_increment,username varchar(255) NOT NULL UNIQUE,hash varchar(255),api_key varchar(255) NOT NULL UNIQUE );
+CREATE TABLE IF NOT EXISTS public_keys (id integer primary key autoincrement, user_id integer NOT NULL, friendly_name varchar(255), pub_key blob NOT NULL);
```

Comment:
Let's create a new database migration for this instead of editing old ones. This makes it easier to keep track of changes that have been applied over time.